Jenkinsfile

```diff
@@ -1,6 +1,10 @@
 pipeline { 
   // use the repository Dockerfile for building the environment image
   agent { dockerfile true }
+  environment { 
+    npm_config_cache = 'npm-cache'
+    HOME="${env.WORKSPACE}"
```

Comment:
Overriding `$HOME` seems like a strange thing to do, but if it solves a problem...

Reply:
Yeah, the `Cypress` dependency was failing install on 
```
----------

Error: Target directory is expected to be absolute
----------
```

This seems to appease it. I think it's something to do with the container internal path and the Docker folder, it will take further looking into.